pkg/queue/cluster_queue_impl.go

```diff
@@ -119,6 +119,7 @@ func (c *clusterQueueBase) PushOrUpdate(wInfo *workload.Info) {
 		// to potentially become admissible, unless the Eviction status changed
 		// which can affect the workloads order in the queue.
 		if equality.Semantic.DeepEqual(oldInfo.Obj.Spec, wInfo.Obj.Spec) &&
+			equality.Semantic.DeepEqual(oldInfo.Obj.Status.ReclaimablePods, wInfo.Obj.Status.ReclaimablePods) &&
```

Comment:
Should we extend this UT: https://github.com/kubernetes-sigs/kueue/blob/1e66b4360a9870136c816372a5818f35b50bf31b/pkg/queue/cluster_queue_impl_test.go#L228

Reply:
added